uploader/src/main/resources/cfn-template.yaml

```diff
@@ -0,0 +1,139 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "A game-changing pipeline for uploading video to the internet"
```

Comment:
Oh oops, that shouldn't have been left in. I'll remove.

:1st_place_medal: ⚾️ Congrats on the results ⚾️ :1st_place_medal: